diff -Nur OpenSim-/Framework/Servers/HttpServer/OSHttpServer/HttpClientContext.cs OpenSim/Framework/Servers/HttpServer/OSHttpServer/HttpClientContext.cs
--- OpenSim-/Framework/Servers/HttpServer/OSHttpServer/HttpClientContext.cs	2020-09-12 15:44:20.595413779 +0900
+++ OpenSim/Framework/Servers/HttpServer/OSHttpServer/HttpClientContext.cs	2020-09-12 15:45:25.078639146 +0900
@@ -133,8 +133,15 @@
                 if (_cert1 != null)
                 {
                     X509Certificate2 _cert2 = new X509Certificate2(_cert1);
-                    if (_cert2 != null) 
-                        SSLCommonName = _cert2.GetNameInfo(X509NameType.SimpleName, false);
+                    if (_cert2 != null)
+                    {
+                        try {
+                            SSLCommonName = _cert2.GetNameInfo(X509NameType.SimpleName, false);
+                        }
+                        catch {
+                            SSLCommonName = "unknownName";
+                        }
+                    }
                 }
             }
 
