diff -Nur OpenSim-/Framework/Servers/HttpServer/BaseHttpServer.cs OpenSim/Framework/Servers/HttpServer/BaseHttpServer.cs
--- OpenSim-/Framework/Servers/HttpServer/BaseHttpServer.cs	2018-08-25 01:15:06.269483109 +0900
+++ OpenSim/Framework/Servers/HttpServer/BaseHttpServer.cs	2018-09-04 18:10:54.009103723 +0900
@@ -157,6 +157,11 @@
             m_port = port;
         }
 
+        public RemoteCertificateValidationCallback CertificateValidationCallback
+        {
+            set { m_certificateValidationCallback = value; }
+        }
+
         private void load_cert(string CPath, string CPass)
         {
             try
